students/ghassan/lesson06/test_new_mailroom.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+import unittest
+from new_mailroom import main_menu, send_thankyou, donors
+
+
+class TestMailroom(unittest.TestCase):
+    def test_main_menu(self):
+        ''' this tests if the function is actually
+        returning the user's input '''
+        users_choice = main_menu()
+        self.assertEqual(users_choice, users_choice)
```

Comment:
In this case, `user_choice` will always equal `users_choice` so this isn't actually testing anything.